src/main/java/com/sergiofah/model/Product.java

```diff
@@ -10,10 +12,12 @@ public class Product implements Serializable {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @JsonIncludeProperties(value = {"id"})
```

Comment:
JSON formatting annotations shall not be in Entity classes. It's necessary to create DTO classes.